utils/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.8
-PKG_RELEASE:=2
+PKG_VERSION:=2017-02-09
```

Comment:
My comment is on the change of version from 1.8 to 2017-02-09. That means that once we ship 1.9 or 1.10 it will be unclear which is newer.